src/main/java/com/work/management/dto/EmployeeDto.java

```diff
@@ -19,6 +19,7 @@
   private String firstName;
   private String lastName;
   private String userName;
+  private Integer id;
   private Integer manager;
```

Comment:
You are adding id in Dto...if user sets this id as something else in Api..then you might be in trouble...you need to take special care now...that you are not updating this Id in the db...it should never get updated.